### PR TITLE
BUGZ-132: Fix avatar loading spheres

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.atlassian.net/browse/BUGZ-132

### CODE REVIEW

File: interface/src/avatar/OtherAvatar.cpp [hunk 1]
```diff
@@ -62,7 +62,7 @@ void OtherAvatar::removeOrb() {
 }
 
 void OtherAvatar::updateOrbPosition() {
-    if (_otherAvatarOrbMeshPlaceholderID.isNull()) {
+    if (!_otherAvatarOrbMeshPlaceholderID.isNull()) {
```
Comment (SamGondelman) [inline interface/src/avatar/OtherAvatar.cpp]: whoops

Comment (SamGondelman) [inline interface/src/avatar/OtherAvatar.cpp]: whoops

File: libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp [hunk 2]
```diff
@@ -1519,18 +1519,19 @@ void Avatar::scaleVectorRelativeToPosition(glm::vec3 &positionToScale) const {
 }
 
 void Avatar::setSkeletonModelURL(const QUrl& skeletonModelURL) {
-    AvatarData::setSkeletonModelURL(skeletonModelURL);
-    if (QThread::currentThread() == thread()) {
+    if (QThread::currentThread() != thread()) {
```
Comment (SamGondelman) [inline libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp]: this isn't the fix but the current code seems wrong

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19539/HighFidelity-Beta-PR15584-3273f46.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19539/HighFidelity-Beta-Interface-PR15584-3273f46.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19539/HighFidelity-Beta-PR15584-3273f46.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19539/HighFidelity-Quest-Beta-PR15584-3273f46.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19539/HighFidelity-Beta-PR15584-3273f46.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19539/HighFidelity-Beta-Interface-PR15584-3273f46.exe

**Pull request merged**